tests/acceptance/testall

```diff
@@ -402,40 +414,40 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP CFENGINE_TEST_OVERRIDE_EXTENSION_LIBR
       # Some states are output by dcs.cf.sub, therefore check for both TEST
       # prefix and dcs.cf.sub prefix.
       ESCAPED_TEST="$(echo "($TEST|dcs.cf.sub)" | sed -e 's/\./\\./g')"
-      if egrep -e "R: .*$ESCAPED_TEST [XS]FAIL" $OUTFILE > /dev/null; then
+      if egrep "R: .*$ESCAPED_TEST [XS]FAIL" $OUTFILE > /dev/null; then
```

Comment:
Ah, so ``-e`` was already there, and so must have worked. Maybe I remembered that wrong, or only pure "grep" doesn't support it. Either way, removing it also works!